pkg/cmd/account/helper/helper.go

```diff
@@ -0,0 +1,38 @@
+package helper
+
+import (
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/client"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/environments"
+	"github.com/briandowns/spinner"
+)
+
+// ResolveNamesOrId takes in an array of names and trys to find an exact match.
+// If a match is found it will return its corresponding ID. If no match is found
+// it will return the name as is, in assumption it is an ID.
+func ResolveEnvironmentNames(envs []string, octopus *client.Client, spinner *spinner.Spinner) ([]string, error) {
+	spinner.Start()
+	envIds := make([]string, 0, len(envs))
+loop:
+	for _, envName := range envs {
+		matches, err := octopus.Environments.Get(environments.EnvironmentsQuery{
+			Name: envName,
+		})
+		if err != nil {
+			return nil, err
+		}
+		allMatches, err := matches.GetAllPages(octopus.Environments.GetClient())
+		if err != nil {
+			spinner.Stop()
+			return nil, err
+		}
+		for _, match := range allMatches {
+			if envName == match.Name {
```

Comment:
I used `strings.EqualFold` in create release for case-insensitivity... felt like the right move for things that people type in by hand🤷 
You may want to consider using it here.